calico/felix/fetcd.py

```diff
@@ -129,6 +149,67 @@ def _periodically_resync(self):
             gevent.sleep(sleep_time)
             self.force_resync(reason="periodic resync", async=True)
 
+    @logging_exceptions
+    def _status_reporting(self):
+        """
+        Greenlet: periodically writes to etcd
+
+        :return: Does not return, unless reporting disabled.
+        """
+        _log.info("Started status reporting thread.")
+        ttl = self._config.REPORTING_TTL_SECS
+        interval = self._config.REPORTING_INTERVAL_SECS
+        _log.info("Config loaded, reporting interval: %s, TTL: %s", interval, ttl)
+        if interval == 0:
+            _log.info("Interval is 0, status reporting disabled.")
+            return
+        while True:
+            try:
+                self.update_felix_status(ttl)
+                _log.info("Status updated.")
```

Comment:
This will log every interval seconds - depending on the value of that interval, this could easily swamp the logs and make it hard to pick out other useful stuff. Should we only log the case where there is an error? If there's a good reason to log the success, I'd at least make it debug level rather than info.